src/spikeinterface/sorters/external/kilosortbase.py

```diff
@@ -208,6 +208,16 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
         if retcode != 0:
             raise Exception(f"{cls.sorter_name} returned a non-zero exit code")
 
+        # Clean-up temporary files
+        if params["delete_tmp_files"]:
+            for temp_file in sorter_output_folder.glob("*.m"):
+                temp_file.unlink()
+            for temp_file in sorter_output_folder.glob("*.mat"):
+                temp_file.unlink()
+            (sorter_output_folder / "temp_wh.dat").unlink()
+        if params["delete_recording_dat"]:
```

Comment:
Here you should check if it exists. For binary recordings, we dont make a copy!

Reply:
Done.